### DISCUSSION TITLE
Link attribute "target" is always removed

### DISCUSSION THREAD

**Discussion by interaid-steffen-bugenhagen** [ueberdosis/tiptap#2369] [General Questions] [answered]
Hello,
we have been trying for some time now to enable the attribute "target" for e.g. target="_blank" for the extension "@tiptap/extension-link", but without success so far.

We had already tried to integrate our own mark, interestingly with the same result. "targets" or other notations work, only the "target" attribute is removed immediately. 

We use the following versions here under nuxt 2.15.5:
```json
{
"nuxt":"^2.15.5",
"@tiptap/extension-character-count":"^2.0.0-beta.24",
"@tiptap/extension-dropcursor": "^2.0.0-beta.25",
"@tiptap/extension-gapcursor": "^2.0.0-beta.34",
"@tiptap/extension-highlight": "^2.0.0-beta.32",
"@tiptap/extension-image": "^2.0.0-beta.24",
"@tiptap/extension-link": "^2.0.0-beta.35",
"@tiptap/extension-table": "^2.0.0-beta.46",
"@tiptap/extension-table-cell": "^2.0.0-beta.20",
"@tiptap/extension-table-header": "^2.0.0-beta.22",
"@tiptap/extension-table-row": "^2.0.0-beta.19",
"@tiptap/extension-text-align": "^2.0.0-beta.29",
"@tiptap/extension-typography": "^2.0.0-beta.19",
"@tiptap/starter-kit": "^2.0.0-beta.168",
"@tiptap/vue-2": "^2.0.0-beta.74"
}
```

Either I am exceptionally stupid in searching or the problem does not seem to be very widespread
Is the problem known, are there any pitfalls or workarounds?

Our previous approach to unlocking more attributes on an element:

```js
import Link from '@tiptap/extension-link'

const CustomLink = Link.extend({
  addOptions() {
    return {
      openOnClick: true,
      linkOnPaste: true,
      autolink: true,
      HTMLAttributes: {},
    }
  },
  addAttributes() {
    // Return an object with attribute configuration
    return {
      // allow class attribute
      ...this.parent?.(),
      ...{
        href: {
          default: null,
        },
        target: {
          default: null,
        },
        class: {
          default: null,
        },
        rel: {
          default: null,
        },
      },
    }
  },
})
export default CustomLink
```

Thank you very much for your time

Edit - some missing info:

We use prismjs version 1.25.0 here to edit the raw HTML, the content of wich is then set via

```js
this.editor.commands.setContent(this.editorhtml, false)
```

in TipTap.

**Comment by philippkuehn** [comment]
The link extensions already comes with support for `target` (defaults to `_blank`) so I’m not sure what you are trying to do. Feel free to create a reproducable codesandbox if you found any issues with that.

**Comment by interaid-steffen-bugenhagen** [comment, marked as answer]
I had an idea yesterday and it has been confirmed.
We use for the display in prismjs upstream the package clean-html in version 1.5.0 which seems to remove only this attribute.

The problem is therefore absolutely out of place here, I apologize for the inconvenience.
